android/docs/engine/KMManager/setLongpressDelay.md

```diff
@@ -4,7 +4,7 @@ title: KMManager.setLongpressDelay()
 
 ## Summary
 
-The `setLongpressDelay()` method sets the number of milliseconds to trigger a longpress gesture as a stored preference.
+The `setLongpressDelay()` method sets the number of milliseconds to trigger a longpress gesture as a stored preference and sends it to the KeymanWeb Engine.
```

Comment:
```suggestion
The `setLongpressDelay()` method sets the number of milliseconds to trigger a longpress gesture, applies to the keyboard, and stores preference.
```

This is an internal implementation detail, and doesn't need to be in the API docs :grin: